src/helpers/getPetitionsRequestParams.js

```diff
@@ -1,14 +1,20 @@
 import calculateParamOffset from 'helpers/calculateParamOffset';
 import sanitizeParamLimit from 'helpers/sanitizeParamLimit';
 
-export default ({ limit, page, city }) => {
+export default ({ limit, page, city, sort }) => {
   const saneLimit = sanitizeParamLimit(limit);
 
+  if (sort === 'supporters') {
+    sort = '-supporters.amount';
+  } else {
+    sort = '-created';
+  }
```

Comment:
I guess for now we only need those two conditions, as the default is the same as “Most recent”